matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/internal/crypto/E2eeSanityTests.kt

```diff
@@ -145,16 +145,15 @@ class E2eeSanityTests : InstrumentedTest {
             delay(3_000)
         }
 
-        // check that messages are encrypted (uisi)
+        // Due to the new shared keys implementation, invited user should be able to decrypt messages
```

Comment:
I would to that as a separate test?
I still want to test that in normal scenario you don't get the past history. Two tests would be better

Reply:
There is also a separate test for that. If we do not change that the test will fail while keys will be shared, so user will be able to decrypt.